src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyInfoX509Data.cs

```diff
@@ -149,11 +155,10 @@ public ArrayList IssuerSerials
 
         public void AddIssuerSerial(string issuerName, string serialNumber)
         {
-            BigInt h = new BigInt();
-            h.FromHexadecimal(serialNumber);
+            BigInteger h = BigInteger.Parse(serialNumber, NumberStyles.AllowHexSpecifier);
             if (_issuerSerials == null)
                 _issuerSerials = new ArrayList();
-            _issuerSerials.Add(new X509IssuerSerial(issuerName, h.ToDecimal()));
+            _issuerSerials.Add(new X509IssuerSerial(issuerName, h.ToString()));
```

Comment:
I think the internal-only `BigInt` class used here before had a `ToDecimal` method but, as far as I can tell, `System.Numerics.BigInteger` doesn't have this method. It looks like `ToString` produces the same result, but I could switch to the `BigInteger`'s `decimal` cast operator if you think that's better.